src/hooks/useLanguages.ts

```
@@ -1,11 +1,24 @@
 import { useFetch } from "./useFetch";
 
+type LanguageInfo = {
+  languageName: string;
+  languageIcon: string;
+};
+
 export const useLanguages = () => {
-  const { data, loading, error } = useFetch<LanguageType[]>(
-    "/consolidated/_index.json"
-  );
+  const { data, loading, error } = useFetch<any[]>("/api/snippets");
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)